internal/injector/builtin/yaml/go-main.yml

```diff
@@ -21,7 +21,36 @@ aspects:
       - prepend-statements:
           imports:
             tracer: gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer
+            profiler: gopkg.in/DataDog/dd-trace-go.v1/profiler
+            log: log
+            os: os
           template: |-
             tracer.Start(tracer.WithOrchestrion(map[string]string{"version": {{printf "%q" Version}}}))
             defer tracer.Stop()
 
+            switch os.Getenv("DD_PROFILING_ENABLED") {
+            case "1", "true", "auto":
+              // The "auto" value can be set if profiling is enabled via the
+              // Datadog Admission Controller. We always turn on the profiler in
+              // the "auto" case since we only send profiles after at least a
+              // minute, and we assume anything running that long is worth
+              // profiling.
+              err := profiler.Start(
+                profiler.WithProfileTypes(
+                  profiler.CPUProfile,
+                  profiler.HeapProfile,
+                  // Non-default profiles which are highly likely to be useful:
+                  profiler.GoroutineProfile,
+                  profiler.MutexProfile,
+                ),
+                profiler.WithTags("orchestrion:true"),
```

Comment:
Yeah, I'll check on the status of that. It should be straightforward to add that info to profiles from the client library since the client library can tell [whether it was built with Orchestrion](https://github.com/DataDog/dd-trace-go/blob/6674e13ea04b053da120c3b83a4d79d99f2d5e5e/internal/orchestrion/orchestrion.go#L8-L16).